app/src/lib/menu-update.ts

```diff
@@ -164,12 +166,29 @@ function getMenuState(state: IAppState): Map<MenuIDs, IMenuItemState> {
     menuStateBuilder.disable('pull')
   }
 
+  const welcomeScopedIds: ReadonlyArray<MenuIDs> = [
```

Comment:
> shouldn't About be accessible no matter what state the app is in? Since it's a standard macOS menu item, I'd say yes.

I'd say yes as well if it actually did anything. As it stands right now we're not able to show dialogs while in the welcome flow. When (if) that changes I agree that it should be enabled.